frontend/test/metabase/scenarios/smoketest/user.cy.spec.js

```diff
@@ -186,8 +190,9 @@ describe("smoketest > user", () => {
     cy.icon("calendar").click();
 
     visualize();
+    cy.wait("@dataset");
```

Comment:
```suggestion
    visualize();
```

`visualize` helper already waits for `dataset`.

Reply:
Good catch, thanks!